src/librustdoc/html/static/main.js

```diff
@@ -1411,6 +1411,14 @@ function getSearchElement() {
             return tmp;
         }
 
+        // In case someone wants to close the current tag to inject some other unwanted things.
+        // For example: "</span><iframe whatever>"
```

Comment:
Given the previous comment how about:

```suggestion
        // Sanitise the incoming HTML a little to improve the chances of it surviving in the <span>
        // for the search results
```

Reply:
:+1: